src/coreclr/src/vm/gchelpers.cpp

```diff
@@ -754,7 +671,7 @@ OBJECTREF AllocateArrayEx(MethodTable *pArrayMT, INT32 *pArgs, DWORD dwNumArgs,
             // cases we care about (single and multi-dim arrays of value types) have an even number of DWORDs
             // in their headers so the alignment requirements for the header and the payload are the same.
             _ASSERTE(((pArrayMT->GetBaseSize() - SIZEOF_OBJHEADER) & 7) == 0);
-            orArray = (ArrayBase *) AllocAlign8(totalSize, flags);
+            orArray = (ArrayBase*)Alloc(totalSize, flags | GC_ALLOC_ALIGN8);
```

Comment:
Same here.

Reply:
(and other places)

Reply:
This is about the 3 places where `GC_ALLOC_ALIGN8` is used, right?

Reply:
Right